Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -21,6 +21,7 @@ var _initialNotification = RCTPushNotificationManager &&
 
 var DEVICE_NOTIF_EVENT = 'remoteNotificationReceived';
 var NOTIF_REGISTER_EVENT = 'remoteNotificationsRegistered';
+var NOTIF_REGISTER_ERROR_EVENT = 'remoteNotificationsRegisteredError';
```

Comment:
This name doesn’t really read nicely. What about a name like `remoteNotificationsRegistrationError` (and in all other places in the code) ?